assets/styles/components/_sidebar-subscriptions.scss

```diff
@@ -59,9 +75,16 @@
         }
       }
 
+      &:last-child {
+        border-bottom: unset;
+      }
+
       .magazine-subscription-avatar {
+        margin: 0;
         height: 1.5rem;
+        max-height: 1.5rem;
         width: 1.5rem;
+        max-width: 1.5rem;
         object-fit: scale-down;
         border-radius: 50%;
         display: inline;
```

Comment:
I think this is overriding the `image-inline` class, so the alt text is overflowing (`image-inline` sets a `inline-block` display)

so it's doing this when the images break
![image](https://github.com/MbinOrg/mbin/assets/146029455/4741d885-004a-4bbb-a378-9eb6fe36fd3a)

rather than
![image](https://github.com/MbinOrg/mbin/assets/146029455/12d861f7-900a-45f0-8116-f09680538d10)

Reply:
you're right, I'll remove that :+1: